src/lib/deeplink/handlers/add-site-blueprint-with-url.ts

```diff
@@ -14,8 +13,8 @@ import { getMainWindow } from 'src/main-window';
  * - wp-studio://add-site?blueprint_url=<encoded-url>
  * - wp-studio://add-site?blueprint=<base64-encoded-json>
  *
- * It either downloads the blueprint from the URL or decodes the base64 blueprint,
- * and opens the Add Site modal with the blueprint pre-filled.
+ * Both types of blueprints are saved to a temporary file, validated,
```

Comment:
```suggestion
 * Both URL and Base64 encode json types of blueprints are saved to a temporary file, validated,
```
I think it would be nice if we specify the names of both types in the comment.

Reply:
I agree. 👍🏼 I have committed the change in https://github.com/Automattic/studio/pull/2103/commits/68d7dc5877a4f32c7395078ccd93e14ccb7cbe11.